nebula_ros/include/nebula_ros/hesai/hesai_ros_wrapper.hpp

```diff
@@ -29,17 +26,16 @@
 #include <rclcpp/rclcpp.hpp>
 #include <rclcpp_components/register_node_macro.hpp>
 
-#include "nebula_msgs/msg/nebula_packet.hpp"
+#include <pandar_msgs/msg/pandar_scan.hpp>
```

Comment:
No need for nebula_packets and nebula_packet? (Ignore if not, I haven't gone through the include tree)

Reply:
Good catch! Re-added in dbd8262